src/components/Pagination/Pagination.js

```diff
@@ -52,7 +52,7 @@ const Pagination = props => {
 
     const handleClick = e => {
         const {
-            dataset: { page = null }
+            dataset: { page }
```

Comment:
why remove this default?

Reply:
I was thinking the default doesn't make much sense because I don't think `data-page` can ever be `undefined` and cannot be affected from outside this component. And I don't see how it can be tested.

Reply:
in some funny edge case through some unforeseen event propagation? anyway, what is the harm in having that fallback?

Reply:
No although it may look like that it wasn't necessarily my intention to reach 100% but in this case looking at coverage can be useful to spot redundant code which I thought would be the case here. But I think you're right that it doesn't do much harm either. I will revert it (if choose to keep some other changes in this branch).